docs/source/get-started.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Getting Started with Apollo iOS
+sidebar_title: Get Started
+---
+
+import SPMXcodeInstallationPanel from "../shared/spm-xcode-installation-panel.mdx"
+import SPMPackageInstallationPanel from "../shared/spm-package-installation-panel.mdx"
+import SPMSetupCodegenPanel from "../shared/spm-setup-codegen-panel.mdx"
+import PodsInstallationPanel from "../shared/pods-installation-panel.mdx"
+import PodsSetupCodegenPanel from "../shared/pods-setup-codegen-panel.mdx"
+import CarthageInstallationPanel from "../shared/carthage-installation-panel.mdx"
+import CarthageRunScriptPanel from "../shared/carthage-run-script-panel.mdx"
+
+Follow the steps below to add Apollo iOS to your app:
+
+## 1. Install the Apollo frameworks
+
+You can install Apollo iOS into your project using any of the three major Cocoa ecosystem package managers: Swift Package Manager, CocoaPods, or Carthage.
+
+<a name="spm-package-installation" />
+<SPMPackageInstallationPanel />
+
+<a name="spm-xcode-installation" />
+<SPMXcodeInstallationPanel />
+
+<a name="cocoapods-installation" />
+<PodsInstallationPanel />
+
+<a name="carthage-installation" />
+<CarthageInstallationPanel />
+
+## 2. Add a schema file to your target directory
+
+For Apollo iOS to generate models for your GraphQL operations, you need a local copy of your GraphQL server's schema. To acquire this schema, see [Downloading a schema](./downloading-schema/).
+
+## 3. [Optional] Install the Xcode add-ons for syntax highlighting
+
+If you would like to add syntax highlighting for GraphQL query document files to Xcode, install the Xcode GraphQL add-ons.
+
+Check out the [`xcode-graphql`](https://github.com/apollographql/xcode-graphql) repository and follow its [installation guide](https://github.com/apollographql/xcode-graphql#installation).
+
+## 4. Create `.graphql` files for your GraphQL operations
+
+Apollo iOS generates code from GraphQL queries and mutations defined in files in your target. In order to use Apollo iOS, you'll need to define at least one operation GraphQL operation.
+
+GraphQL operation and fragment definitions traditionally have the file extension `.graphql`. The generated models will have the file extension `.graphql.swift`.
+
+## 5. Setup and run code generation
+
+Apollo iOS code generation uses your `.graphql` files to generate API code that helps you execute all forms of GraphQL operations, as well as parse and cache operation responses.
+
+Whenever you make changes to your GraphQL operation definitions, you'll need to run the code generation engine to re-generate your GraphQL models.
+
+The easiest way to do this is with the Codegen CLI provided with Apollo iOS.
+
+> For more advanced usage and configuration (including use with modularized projects), see [Code Generation](./code-generation).
+>
+> To use Apollo's code generation and schema downloader from within any Swift script or library, check out [Swift Scripting](./swift-scripting).
+
+<a name="spm-run-codegen" />
```

Comment:
Something weird is happening with this first `ExpansionPanelListItem` in this callout — it's not wrapping correctly or allowing scrolling, so half of the content isn't accessible or visible.

Reply:
Good  catch. This is a bug for @trevorblades I think. Any chance we can get this fixed quickly Trevor?

If you see this image, the text only in the first list item is not wrapping correctly and text is getting cut off. Also the code blocks in the list item does not allow scrolling so you can't see the rest of the code block!

The second list item is working correctly though.

![image](https://user-images.githubusercontent.com/6243461/193084285-be145a8e-c078-4165-aeb4-ce867103b775.png)